src/main/java/com/openlattice/scrunchie/search/ConductorElasticsearchImpl.java

```diff
@@ -1294,7 +1295,14 @@ public SearchResult executeEntitySetMetadataSearch(
                 .actionGet();
 
         List<Map<String, Object>> hits = Lists.newArrayList();
-        response.getHits().forEach( hit -> hits.add( hit.getSourceAsMap() ) );
+        response.getHits().forEach( hit -> {
+            Map<String, Object> entitySetResult = hit.getSourceAsMap();
+            if ( excludePropertyTypes ) {
+                entitySetResult.remove( PROPERTY_TYPES );
```

Comment:
I'm not familiar with how elasticsearch is setup but I'm guessing its nontrivial to have the elasticsearch result exclude property types, is that accurate?

Reply:
there *are* ways to do it, but the whole entity set metadata indexing piece is pretty ugly and implementing this will be pretty ugly either way, so I just want to rewrite a good chunk of this at some point